scripts/monitoring/cron-send-etcd-status.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
You should do a '#!/bin/bash -e'  so that it exits on error and ops-runner will alert us to the problem. Otherwise this script could fail and still exit with an exit code of 0.